template.php

```diff
@@ -122,9 +122,58 @@ function europa_dropdown(array $variables) {
  */
 function _europa_array_find($needle, array $haystack) {
   foreach ($haystack as $key => $value) {
-    if (FALSE !== stripos($value, $needle)) {
+    if (is_string($value) && FALSE !== stripos($value, $needle)) {
       return $key;
     }
   }
   return FALSE;
 }
+
+/**
+ * Delete all elements in the needle into haystack.
+ *
+ * @param array $needle
+ *   The array of string to search for.
+ * @param array $haystack
+ *   The array to search in.
+ */
+function _europa_array_delete(array $needle, array &$haystack) {
+  array_filter($haystack, function($class, $key) use (&$needle, &$haystack) {
```

Comment:
`&$needle` needs to be replaced by `$needle`

Btw, can't we just use `array_diff()` for this ?